ui/src/frontend/track_group_panel.ts

```diff
@@ -211,8 +211,7 @@ export class TrackGroupPanel extends Panel<Attrs> {
     const shell = assertExists(dom.querySelector('.shell'));
     this.shellWidth = shell.getBoundingClientRect().width;
     // TODO(andrewbb): move this to css_constants
-    this.backgroundColor =
-          getComputedStyle(dom).getPropertyValue('--collapsed-background');
+    this.backgroundColor = this.trackGroupState.collapsed ? getComputedStyle(dom).getPropertyValue('--collapsed-background'): getComputedStyle(dom).getPropertyValue('--expanded-background');
```

Comment:
I'm sure that ESLint will complain about this line being too long. 😀

Reply:
It didn't ,but I'll spread it out